README.md

```diff
@@ -31,16 +31,91 @@ If you don't have `cargo` or `npm` installed, you will need to follow these [add
 
 For information regarding updating Wrangler, click [here](https://workers.cloudflare.com/docs/quickstart/updating-the-cli/).
 
+## Getting Started
+
+Once you have installed Wrangler, spinning up and deploying your first Worker is easy!
+
+```console
+$ wrangler generate my-worker
+$ cd my-worker
+# update your wrangler.toml with your Cloudflare Account ID
+$ wrangler config # => enter your api key and email
+$ wrangler publish
+```
+
+## 🎙️ Top Level Commands
+
+- ### 👯 `generate`
+
+  Scaffold a project, including boilerplate code for a Rust library and a Cloudflare Worker.
+
+  ```bash
+  wrangler generate <name> <template> --type=["webpack", "javascript", "rust"]
+  ```
+
+  All of the arguments and flags to this command are optional:
+  
+  - `name`: defaults to `worker`
+  - `template`: defaults to the [`https://github.com/cloudflare/worker-template`](https://github.com/cloudflare/worker-template)
+  - `type`: defaults to ["webpack"](https://github.com/cloudflare/wrangler/blob/master/docs/content/webpack.md)
```

Comment:
Find and replace all links should be:

https://github.com/cloudflare/wrangler/blob/master/docs/content => developer.cloudflare.com


```
https://github.com/cloudflare/wrangler/blob/master/docs/content/webpack.md => https://developers.cloudflare.com/workers/wrangelr/webpack
https://github.com/cloudflare/wrangler/blob/master/docs/content/kv_commands.md => https://developers.cloudflare.com/workers/wrangelr/kv_commands
...
https://github.com/cloudflare/wrangler/blob/master/docs/content/environments.md => https://developers.cloudflare.com/workers/wrangelr/configuration/environments
```

Reply:
🤦‍♀